src/usr/local/www/system.php

```diff
@@ -506,7 +526,7 @@ function is_timezone($elt) {
 		$dnsserver
 	))->setHelp(($is_last_dnsserver) ? $dnsserver_help:null);
 
-	if ($multiwan)	{
+	if ($multiwan > 1)	{
```

Comment:
Replace TAB by space between `)` and `{`

Reply:
Fixed